### DISCUSSION TITLE
Relocating Error

### DISCUSSION THREAD

**Discussion by cmcd500** [rshea0/enjinn#79] [Help]
Hi There, 

the program is pretty easy to use which is great,  I am getting this error while trying to relocate the track.  I just wanna make sure I am doing this right.  My Engine Library is saved on my D drive Along with all my music.  I typically buy some tracks add cues and loops put it in a little mix to get used to the tracks. Then from there I separate them to where I would normally want to play within a set.  
So I assuming it should work as tracks I buy are in a beatport folder.  I then lets say afterwards I move it the track a folder called start.  I have changed the path settings in tuneup PRIME to the "d" drive and try to relocate the track and this is the error

update 'track' set 'path' = '..Folder Path/filename.mp3 where'id' = 9606 - SQLITE_CONSTRAINT= UNIQUE CONTRAINT FAILED: track.path

Thanks for your help in advance
Colin

**Comment by shayded-exe** [comment]
It looks like you might already have the new file in your library. Look in your Engine Library to see if the file in the start folder is already imported.

**Comment by cmcd500** [comment]
Thanks Ryan for getting back to me. 

 Let me ask this then.  So if I move the track within my computer directly, do I just run the program and it will automatically update engine prime library?

I think I removed the folder from EP library, then  imported the update folder with the tracks moved back into EP

Thanks Colin

**Reply by shayded-exe** [reply]
Correct. If you move the file, Tuneup will relocate it and update your library. The original issue you were having looks to be because you had two entries for the same track but at different paths.

**Reply by cmcd500** [reply]
Alright so I did the above and now I get this error.  Sorry

COMMIT; - SQLITE_IOERR: disk I/0 error

**Reply by shayded-exe** [reply]
Do you have Engine open when you run it? Engine needs to be closed first.

**Comment by cmcd500** [comment]
Awesome Thanks worked like a charm. 

So some of the tracks have changed location when done wrong always come up in the relocate section. Anyway to hide or delete the missing tracks from tune up so you don't want to see them anymore?

**Reply by shayded-exe** [reply]
Glad it worked! And yeah I'm planning on adding a delete function to relocate

**Comment by cmcd500** [comment]
Thanks again Ryan for your help!!
